rasa/shared/utils/io.py

```diff
@@ -314,7 +315,7 @@ def env_var_constructor(loader, node):
             w for w in expanded_vars.split() if w.startswith("$") and w in value
         ]
         if not_expanded:
-            raise ValueError(
+            raise RasaException(
                 "Error when trying to expand the environment variables"
                 " in '{}'. Please make sure to also set these environment"
                 " variables: '{}'.".format(value, not_expanded)
```

Comment:
More of a style comment, shouldn't this be updated to a f-string according to Notion [code convention](https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#208f2a9207304db8b473ac348a55d26f) ?

Reply:
good catch!